receiver/purefareceiver/README.md

```diff
@@ -0,0 +1,39 @@
+# Pure Storage FlashArray Receiver
+
+| Status                   |                     |
+| ------------------------ |---------------------|
+| Stability                | [in-development]    |
+| Supported pipeline types | metrics             |
+| Distributions            | [contrib]           |
+
+The Pure Storage FlashArray receiver, receives metrics from Pure Storage internal services hosts.
+
+Supported pipeline types: metrics
+
+## Configuration
+
+The following settings are required:
+ -  `endpoint` (default: `http://172.0.0.0:9490/metrics/array?endpoint=gse-array01`): The URL of the scraper selected endpoint
+ 
+Example:
+
+```yaml
+receivers:
+  purefa:
+    endpoint:
+    arrays:
+      address:
+      token:
+    settings:
```

Comment:
Giving some more thought to this, how about we have this:

```yaml
settings:
  reload_intervals:
    array: 1s
    host: 30s
    volume: 1m
    pods: 1s
    directories: 15s
```

(the values are just placeholders)